lsp/tsconfig.json

```diff
@@ -30,11 +30,17 @@
         {
             "path": "./server/aws-lsp-cloudformation"
         },
+        {
+            "path": "./server/aws-lsp-s3"
+        },
         {
             "path": "./app/aws-lsp-buildspec-binary"
         },
         {
             "path": "./app/aws-lsp-cloudformation-binary"
+        },
+        {
+            "path": "./app/aws-lsp-s3-binary"
```

Comment:
Looks like many projects. And the size of the package-lock.json in this PR is a hint about the dependency explosion we will need to manage with all of these micro-projects.

Reply:
The cool thing about how an npm monorepo works is that there is a single package-lock.json (and single node_modules) folder that serves as the composite for all projects in the monorepo. Each project still uses what it needs (without bundling the other packages that it doesn't reference), but you don't have redundant node_modules folders for each project.

Reply:
Nice!